package.json

```diff
@@ -31,12 +31,23 @@
     "npm": "3.3.5"
   },
   "dependencies": {
-    "babel-core": "^5.8.25",
-    "babel-eslint": "^4.1.3",
-    "babel-loader": "^5.3.2",
     "bluebird": "^2.10.2",
-    "chai": "^3.3.0",
     "classnames": "^2.1.5",
+    "jquery": "^2.1.4",
+    "lodash": "^3.10.1",
+    "react": "^0.14.0",
+    "react-dom": "^0.14.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.3.17",
+    "babel-core": "^6.3.2",
+    "babel-eslint": "^4.1.6",
+    "babel-loader": "^6.2.0",
+    "babel-plugin-add-module-exports": "^0.1.2",
+    "babel-preset-es2015": "^6.1.18",
+    "babel-preset-react": "^6.1.18",
+    "babel-preset-stage-1": "^6.1.18",
+    "chai": "^3.3.0",
```

Comment:
Pulled runtime deps into dependencies, all others are devDependencies.  Also, bumped us down to stage 1 for a bit more stability.